src/style.css

```diff
@@ -0,0 +1,32 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  background-color: white;
+  position: relative;
+}
+
+.yellow {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-position: center center;
```

Comment:
check background for the each star, right now background image is repeated, fix it